.travis.yml

```diff
@@ -8,3 +8,5 @@ go:
 
 script:
  - go test -short ./...
+ - go tool vet .
+ - diff -u <(echo -n) <(gofmt -d .)
```

Comment:
Other Prometheus repos just use `gofmt -d | grep '^'` (via Makefile.common, but we don't have a Makefile here as it is a library). I think it's better to do the fmt check in the same way everywhere.

Reply:
The plan is to include the Makefile in all the Go repos, including this one. Not all of it will make sense obviously, but at least these things will be consistent.

@krasi-georgiev

Reply:
@beorn7 gofmt -d . alone will not work as it seems that this command return 0 (error_code 0) even if the code is not properly formatted. "man diff" says "Exit status is 0 if inputs are the same, 1 if different, 2 if trouble." that's what we leverage here.

@brian-brazil I am all in favor a makefile but my makefile mafic sucks. In the mean time, this PR could be a step in the right direction.

Reply:
why not use the `style` target of the common makefile?
https://github.com/prometheus/prometheus/blob/master/Makefile.common


only targets that use the `pkgs          = ./...` will fail when used with golang prior the 1.9 version
https://github.com/prometheus/prometheus/blob/4a4e8a7d3bab177aa446e8b0fd4de2c41bf5298d/Makefile.common#L34